OasisDEX/oasis-borrow
# Lw/6263 extract existing trigger data

# [Title](https://shortcutLink)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
Create ATP trigger go back to optimization tab after it is added
- `console.log(decodedTriggers)` in `getTriggersByType` verify if everything is as expected 

![image](https://user-images.githubusercontent.com/6871923/194579106-6a0bea1e-0638-4237-860e-194a4b176fd6.png)
